### DISCUSSION TITLE
Database blocked after 10 seconds

### DISCUSSION THREAD

**Discussion by thewalkerz** [Kunzisoft/KeePassDX#1339] [Q&A]
I have a question, when I try to enable or disable database lock timeout, the app ignores that and locks after 5 seconds when I open a entry, this is a bit annoying as it doesn't give me time when I try to type the password manually. This is an active issue or I am doing something wrong?

Thanks for this password manager I love it too much. 😊

**Comment by J-Jamet** [comment]
https://github.com/Kunzisoft/KeePassDX/wiki/FAQ#why-did-the-app-lock-before-the-timeout-duration-has-elapsed

;)
